### ISSUE TITLE
support estimate types different from `MAP`? 

### ISSUE THREAD

**Issue by otizonaizit** [wichmann-lab/python-psignifit#92]
The Matlab version, and partially the master branch are supporting configuration['estimate_type'] different from `MAP`: `mean` and `MLE`. Do we want to support them? Felix says there's no need.

**Comment by otizonaizit** [comment]
by the way `MAP` and `MLE` are exactly the same, and `mean` does not make sense: psignifit hasn't been even tested to work well with that estimate type

**Comment by pberkes** [comment]
If implemented correctly, MAP should use the prior, while MLE should not

**Comment by HeikoSchuett** [comment]
MLE and MAP do indeed result in the same behaviour in MATLAB. We got to many divergent results with the actual MLE to be useful.

The mean is sensible though and we did test it. It performs a little worse than the MAP, but as far as I know Bayesian stats usually considers the posterior mean to be a better summary.
Also computing the posterior mean once you have the marginals is ~5 lines of code, so it does not really seem like the place to cut...
